demos/python_demos/formula_recognition_demo/requirements.txt

```diff
@@ -0,0 +1 @@
+sympy
```

Comment:
This can't be the only dependency. Pretty sure I've also seen `tqdm`.

Reply:
Added `tqdm`. Not sure if I should also add `numpy` & `opencv-python`

Reply:
I think adding `numpy` makes sense. However, `opencv-python` should not be added, because the demo is supposed to work with the version of OpenCV in the OpenVINO toolkit.

Reply:
Added numpy